components/blocks/About.vue

```diff
@@ -11,12 +11,13 @@
         class="content-section"
       >
         <h2 class="section-title title">
-          <fa
-            size="2x"
-            class="mr-3 has-text-white"
-            :icon="[item.fa.prefix, item.fa.icon]"
+          <span
```

Comment:
Why drop using `<fa/>?

Reply:
I guess, dropping it lets us change to any other icon library? Looking through what our site uses, the font awesome free is pretty limiting. https://heroicons.com may have more options

Reply:
Or we go with material icons. They are plentiful.

Reply:
I had two main thoughts: 1) with the `<fa>` icons (`@nuxt/fontawesome`) we're styling svg, so we're away from base bulma for controling those styles; 2) it seems like the main benefit of using @nuxt/fontawesome is the tree shaking so that you only import/bundle what you need, but we can't do that since the icons are determined by the content, not the code in many cases